### ISSUE TITLE
Remove the light theme of the Raft Site

### ISSUE THREAD

**Issue by victoriaatraft** [raft-tech/raft-tech.github.io#494]
**Currently:**
There are accessibility concerns with the white theme and we need to remove it until we prioritize optimizing said version. Currently it is not accessible/WCAG compliant and not polished.

In addition:
- Please remove the sun/moon icon in the header
- Remove "Accessibility matters" And the options in the **navigation**
- Remove "Accessibility matters" And the options in the **Footer**
